www/main/overlay/utils/get-theme-markup.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import convertToCamelCase from '../../common/utils/convert-to-camel-case';
+
+const electron = require('electron');
+
+const { remote } = electron;
+const { i18n } = remote.require('./app');
+
+const getThemeMarkup = (themeObjects, handleThemeChange) =>
+  Object.keys(themeObjects).map(theme => {
+    const currentTheme = themeObjects[theme];
+    const themeClass = i18n
+      .t(`THEMES.${currentTheme.label}`)
+      .toLowerCase()
+      .split(' ')
+      .join('-');
+
+    return (
+      <div
+        key={`theme-${theme}`}
+        className={`overlay-theme-swatch`}
+        data-theme-name={convertToCamelCase(themeClass)}
```

Comment:
This attribute is used to get the theme name when clicked on the div.

```javascript
  const handleThemeChange = e => {
    const clickedTheme = e.currentTarget.dataset.themeName;
    ...
}
```

React allows and prefers the data attributes to be in this case instead of camel case.
https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html#data-and-aria-attributes